README.md

```diff
@@ -77,12 +84,30 @@ Keep in mind that both the frontend and backend have hot reload enabled so you w
 
 Following the starter-code's pattern, complete these tasks:
 
-Currently, our restaurant directory maintains a singular list of restaurants. Suppose we now want the ability to create restaurant groups (e.g. "Uni Plaza", "Cafes", "Open Late", etc.). Each group consists of a subset of the restaurants; a restaurant can be part of multiple groups.
+Currently, our restaurant directory maintains a singular list of restaurants. 
+Suppose we now want the ability to create restaurant groups (e.g. "Uni Plaza", "Cafes", "Open Late", etc.). 
 
-A `RestaurantGroup` has these fields: id, name, description, restaurantIds
+A single group can contain multiple restaurants, and a single restaurant can be part of multiple groups.
 
-1. Create REST endpoints for `RestaurantGroup`, supporting CRUD operations
-    * The API responses should include full restaurant information, not just the ids
-    * Deleting a restaurant should result in its removal from all groups it was part of
+A `RestaurantGroup` has these fields: `id`, `name`, `description`, `restaurantIds`
 
+1. Using the existing code as a template, create REST endpoints for `RestaurantGroup`, supporting C.R.U. (Create, Retrieve, Update) operations. 
```

Comment:
Look like update got moved to Extensions?

Reply:
Good catch!